yb-voyager/cmd/exportData.go

```diff
@@ -312,6 +311,20 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 		if err != nil {
 			return fmt.Errorf("failed to determine if Oracle JDBC wallet location is set: %v", err)
 		}
+	} else if source.DBType == "yugabytedb" {
+		config.YBServers = source.DB().GetServers()
+		if startClean {
+			config.YBStreamID, err = dbzm.GenerateAndStoreYBStreamID(*config)
```

Comment:
You should change the `GenerateAndStoreYBStreamID()` to accept pointer to `Config`. That way you won't need to send config by-value here.